crates/turbopack-dev-server/src/http.rs

```diff
@@ -117,13 +121,21 @@ pub async fn process_request_with_content_source(
                 }
 
                 let content = file.content();
-                header_map.insert(
-                    "Content-Length",
-                    hyper::header::HeaderValue::try_from(content.len().to_string())?,
+                // Grab ropereader stream, coerce anyhow::Error to std::io::Error
+                let stream_ext = content
+                    .read()
+                    .into_stream()
+                    .map_err(|err| std::io::Error::new(std::io::ErrorKind::Other, err));
+
+                let gzipped_stream = tokio_util::io::ReaderStream::new(
+                    async_compression::tokio::bufread::GzipEncoder::new(
+                        tokio_util::io::StreamReader::new(stream_ext),
```

Comment:
Nit: with `use tokio_util::io::{ReaderStream, StreamReader}; use async_compression::tokio::bufread::GzipEncoder;`
```suggestion
                let gzipped_stream = ReaderStream::new(
                    GzipEncoder::new(
                        StreamReader::new(stream_ext),
```